packaging/windows/sharedframework/sharedframework.wxs

```diff
@@ -11,6 +11,27 @@
         <Feature Id="MainFeature" Title="Main Feature" Level="1">
             <ComponentGroupRef Id="InstallFiles" />
             <ComponentGroupRef Id="AuthoredRegistryKeys"/>
+            <Component Id="BreadcrumbStoreFolder" Directory="BreadcrumbStore" Guid="{DA957490-EA59-42B9-B9D6-3E80F6FB0ED0}">
```

Comment:
Components and their GUIDs are the thing I dislike *most* about the Windows Installer. Ever since I was an intern in 1998... now I just feel old.